src/modules/MappingUtils.ts

```diff
@@ -1,14 +1,15 @@
 import { set } from 'lodash'
 import { CaasApi_Item, MappedCaasItem } from '../types'
-import { ResolvedReferencesInfo, ReferencedItemsInfo, CaaSMapper } from './CaaSMapper'
+import { ResolvedReferencesInfo, ReferencedItemsInfo } from './CaaSMapper'
 
-const getItemId = (item: MappedCaasItem | CaasApi_Item): string => {
+const getItemId = (item: MappedCaasItem | CaasApi_Item, remoteProjectId?: string): string => {
+  const projectPrefix = remoteProjectId ? `${remoteProjectId}#` : ''
```

Comment:
This format with <projectId>#<itemId> - is that something we came up with ourselves and is fsxa-api internal or is that based on some other api?
I am trying to understand why it is the way it is, couldn't we just carry the remote project information in a separate variable?

Reply:
The whole mapping process identifies Documents per string id. 
It was first based on uuid solely, which leads to clashes across projects. Hence this "${projectId}#${documentId}" string. How to combine those to make a unique string is arbitrary at that point

Reply:
I understand this is due to historical reasons.